README.md

```diff
@@ -25,3 +25,18 @@ To install third-party libraries, first `cd function`, then `npm install`. You w
 1. `cd function` - make sure you're in the right directory. The root of this repostory is NOT the right directory!
 2. `npm install` - get all the third-party dependencies, if you have not done so already. `npm i` is the shorthand.
 3. `npm run lint` - note you need the extra `run` command.
+
+# Broad Internal
+This section applies only to running the workbench-tos codebase within the Broad Institute.
+
+## Deploying
+To deploy the TOS API:
+1. If you are deploying a new version of the codebase, create a tag or release within github for the commit you want to deploy. By convention, we only deploy tagged commits; we do not care about branches of this repo for deploys. If you simply need to deploy/redeploy a pre-existing tag/release, you can skip this step.
```

Comment:
Is there a naming convention you'd like others to follow?

Reply:
nope